site/data/dynamo.ts

```diff
@@ -67,6 +66,29 @@ export const getPublishedDisruptionsDataFromDynamo = async (id: string): Promise
     return disruptionIds?.map((id) => collectDisruptionsData(dbData.Items || [], id)).filter(notEmpty) ?? [];
 };
 
+export const getSubmittedDisruptionsDataFromDynamo = async (id: string): Promise<Disruption[]> => {
```

Comment:
We don't want to write to the siri table in the case that the disruption is pending approval